uber/templates/preregistration/form.html

```diff
@@ -27,14 +27,14 @@
                 }
             });
         {% endif %}
```

Comment:
Now that I have a chance to double-check, I don't think using ``c.PREREG_DONATION_OPTS`` would work:

```python
>>> c.PREREG_DONATION_OPTS
[(0, 'No thanks'), (20, '+ $20: Shirt'), (50, '+ $50: Supporter Pack'), (85, '+ $85: Food')]
>>> c.DONATION_TIERS
{0: 'No thanks', 50: 'Supporter Pack', 20: 'Shirt', 85: 'Food'}
```

With that being said, checking availability is a good idea, we'll just need to do it differently.